setup.py

```diff
@@ -18,7 +18,7 @@
     ],
     entry_points={
         'flake8.extension': [
-            'M90 = mutable_defaults:MutableDefaultChecker',
+            'M5 = mutable_defaults:MutableDefaultChecker',
```

Comment:
http://flake8.pycqa.org/en/3.5.0/plugin-development/registering-plugins.html says "if the plugin reports only the error code X101 your entry-point would look like: `X101 = flake8_example.ExamplePlugin`", so imo this should be `511`, since that is the only code currently reported, c.f. https://github.com/ebeweber/flake8-mutable/blob/4e0394e61841edff876301a98f6153805ecc4b9b/mutable_defaults.py#L25